client/post-editor/editor-diff-viewer/style.scss

```diff
@@ -1,41 +1,46 @@
-.editor-diff-viewer__title {
-	padding-left: 10px; /* Intentionally non-standard to align editor body margin (10px) */
-	padding-right: 10px;
-	font-family: $serif;
-	font-size: 28px;
-	color: $gray-dark;
-	font-weight: 600;
+/** @format */
+.editor-diff-viewer {
+	margin: 0;
+	position: absolute;
+		top: 120px;
+		bottom: 72px;
+		left: 0;
+	width: 100%;
+	box-sizing: border-box;
+	padding: 24px 32px 32px 32px;
+	overflow-y: auto;
 
-	@include breakpoint( '>480px' ) {
-		font-size: 32px;
+	@include breakpoint( ">660px" ) {
+		top: 0;
+		width: calc( 100% - 230px );
 	}
-}
 
-.editor-diff-viewer {
-	margin: 0 auto;
-	max-width: 700px;
-	width: 100%;
+	.editor-diff-viewer__title {
+		font-family: $serif;
+		font-size: 32px;
+		color: $gray-dark;
+		font-weight: 600;
+		margin: 0 0 24px 0;
+		height: auto;
+		line-height: 1.2;
+	}
 }
 
 .editor-diff-viewer__content {
-	padding: 11px;
+	font-family: $serif;
+	font-size: 15px;
 	white-space: pre-wrap;
-	// following css is a reset from the general <pre> rule that matches
+	padding: 0;
 	background: transparent;
-	font-family: inherit;
-	font-size: inherit;
-	line-height: inherit;
-	margin-bottom: inherit;
-	overflow: auto;
-}
 
-.editor-diff-viewer__additions {
-	background: lighten( $blue-wordpress, 58% );
-	border-bottom: 2px solid $blue-wordpress;
-}
+	.editor-diff-viewer__additions {
+		background: lighten($blue-wordpress, 58%);
```

Comment:
spaces inside brackets

Reply:
Prettier is doing this automatically. Do I need to reconfigure something?

Reply:
if it is automatic, ignore me :) but we do prefer spaced brackets even in CSS, right?

Reply:
> but we do prefer spaced brackets even in CSS, right?

It [looks like we do](https://github.com/Automattic/wp-calypso/blob/27397c4b80026609c0515988bc6b73314f6a7fe6/docs/coding-guidelines/css.md#spacing).  I'll revert these lines, but it does seem like [our fork of prettier](https://github.com/Automattic/calypso-prettier) needs fixing here.